draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -373,8 +373,11 @@ then sends a `CERTIFICATE_NEEDED` frame for stream zero referencing the request,
 indicating that the connection cannot be used for that origin until the
 certificate is provided.
 
-If the server does not have the desired certificate, it MUST send an empty
-`USE_CERTIFICATE` frame for stream zero.  In this case, or if the server has not
+If the server does not have the desired certificate, it MUST send an Empty
```

Comment:
Why only one side?

Reply:
This section is talking specifically about clients probing servers.  But you're right -- there should be a corresponding statement about clients.